2-mandatory/3-financial-times.js

```diff
@@ -5,7 +5,14 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
-    // TODO
+  const rightTitles = [];
+
+  for (titles of allArticleTitles) {
+    if (titles.length <= 65) {
+      rightTitles.push(titles);
+    }
+    return rightTitles;
```

Comment:
It looks like this return statement is not in the right place. You're solution is very close to being correct, can you think of how to fix it?